sdk/tests/conformance/extensions/ext-frag-depth.html

```diff
@@ -116,6 +116,9 @@
 ext = null;
 runAllTests();
 
+// Run deferred link tests.
+runDeferredLinkTests();
```

Comment:
This is an excellent point -- the new runDeferredLinkTests function assumes the extension is supported. It's perfectly legal for it to not be. Adding this to the list of tests in runAllTests() in this and the other extension tests will implicitly fix this problem.